### PR TITLE
Remove Fix from logicalplan

### PR DESCRIPTION
**Pull request by alissapajer**

### CODE REVIEW

File: foundation/src/main/scala/quasar/contrib/matryoshka/EqualT.scala [hunk 1]
```diff
@@ -18,21 +18,27 @@ package quasar.contrib.matryoshka
...
+  def recursive[T[_[_]]: Recursive]: EqualT[T] = new EqualT[T] {
+    def equal[F[_]: Functor]
+      (tf1: T[F], tf2: T[F])
+      (implicit del: Delay[Equal, F]) =
+      del(equalT[F]).equal(tf1.project, tf2.project)
+    }
+
+  implicit val fix: EqualT[Fix] = recursive
+  implicit val mu:  EqualT[Mu]  = recursive
+  implicit val nu:  EqualT[Mu]  = recursive
```
Comment (wemrysi) [inline foundation/src/main/scala/quasar/contrib/matryoshka/EqualT.scala]: Should be `EqualT[Nu]`

Comment (alissapajer) [inline foundation/src/main/scala/quasar/contrib/matryoshka/EqualT.scala]: Nice catch thanks!

**Comment by alissapajer** [comment]
@wemrysi change pushed - back to you!

**Comment by alissapajer** [comment]
Also, I must say how much I love git worktree in this situation. I was able to checkout this branch and run a full compile in the background without interrupting my other work. Truly a great feature.

**Pull request merged**